cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,104 @@
+= CloudHub VPC to CloudHub 2.0 Private Space Upgrade  
+
+//Overview
+To migrate your workloads from CloudHub to CloudHub 2.0, you need to set up your CloudHub 2.0 infrastructure and network connections. One approach is to create new CloudHub 2.0 private spaces with configurations that differ from your existing CloudHub setup. 
```

Comment:
I'm not sure what the second sentence means.

Are you saying:
If you don't need to keep your existing configuration, just create a new CH2 private space.